### PR TITLE
Merge cqc pir clean data into ind cqc data

### PR DESCRIPTION
**Pull request by james-francis-MT**
# Description
Merge PIR data into cqc loc data

# How to test
tested in my stack here: https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/jf-merge_ind_cqc_data_job/runs

https://trello.com/c/DaqhTVRI/58-epic-7-merge-cqc-pir-clean-into-merge-ind-cqc-must

# Developer checklist
- [x] Unit test
- [x] Linked to Trello ticket
- [ ] Documentation up to date

### CODE REVIEW

File: jobs/merge_ind_cqc_data.py [hunk 1]
```diff
@@ -92,6 +106,27 @@ def filter_df_to_independent_sector_only(df: DataFrame) -> DataFrame:
...
+        CQCPIRClean.location_id, CQCLClean.location_id
+    ).withColumnRenamed(CQCPIRClean.care_home, CQCLClean.care_home)
+
+    ind_df_with_pir_data = ind_df_with_pir_import_date.join(
+        formatted_pir_df,
+        [CQCPIRClean.cqc_pir_import_date, CQCLClean.location_id, CQCLClean.care_home],
+        "left",
+    )
+
+    return ind_df_with_pir_data.drop(CQCPIRClean.cqc_pir_import_date)
```
Comment (garypolzin) [inline jobs/merge_ind_cqc_data.py:127]: Can you remove this drop please, handy to have this column in the file so we can see which PIR data was matched in to the file

File: tests/unit/test_merge_ind_cqc_data.py [hunk 2]
```diff
@@ -41,7 +41,10 @@ def setUp(self) -> None:
         )
 
     @patch("jobs.merge_ind_cqc_data.join_ascwds_data_into_merged_df")
-    @patch("jobs.merge_ind_cqc_data.filter_df_to_independent_sector_only")
+    @patch(
+        "jobs.merge_ind_cqc_data.filter_df_to_independent_sector_only",
+        wraps=job.filter_df_to_independent_sector_only,
+    )
```
Comment (garypolzin) [inline tests/unit/test_merge_ind_cqc_data.py:47]: I haven't seen this before, I don't suppose you can explain what this is/why it's needed please? :)

**Review by garypolzin — CHANGES_REQUESTED** [review]

File: tests/unit/test_merge_ind_cqc_data.py (hunk 2 above)
Comment (james-francis-MT) [reply tests/unit/test_merge_ind_cqc_data.py:47]: It keeps the behaviour of the actual function - previously the mock would just return a None object but now it actually runs. You can still make assertions on it though as it is still a mock object

Comment (garypolzin) [reply tests/unit/test_merge_ind_cqc_data.py:47]: ah that's interesting, thanks!

**Review by garypolzin — APPROVED** [review]
looks great thanks

**Pull request merged**